### PR TITLE
Add Azure CLI steps for AKS cluster  creation for Azure BYOK

### PR DESCRIPTION
**Pull request by ritu-rubrik**

### CODE REVIEW

File: azure/onboarding/byok8s/manualsteps/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Documentation for Manually Creating an AKS Cluster for RSC Exocompute
+This documentation outlines the steps to manually create an AKS cluster and associated resources on Azure to be used as n exocompute cluster with RSC
+
+## Prerequisites:
+1. Azure CLI configured with necessary permissions.
```
Comment (arnavrupde1) [inline azure/onboarding/byok8s/manualsteps/README.md:5]: Should we explcitly mention that user needs to run `az login` command and select appropriate subscription before running following commands?

Comment (ritu-rubrik) [reply azure/onboarding/byok8s/manualsteps/README.md:5]: sure will add that

File: azure/onboarding/byok8s/manualsteps/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Documentation for Manually Creating an AKS Cluster for RSC Exocompute
+This documentation outlines the steps to manually create an AKS cluster and associated resources on Azure to be used as n exocompute cluster with RSC
```
Comment (jainmilind) [inline azure/onboarding/byok8s/manualsteps/README.md]: to be used as an exocompute

File: azure/onboarding/byok8s/manualsteps/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+az aks create \
+  --resource-group $RESOURCE_GROUP \
+  --name $CLUSTER_NAME \
+  --location $LOCATION \
+  --node-count 1 \
+  --nodepool-name rubrikcloud \
+  --node-vm-size Standard_E8s_v5 \
+  --enable-private-cluster \
+  --service-principal $SP_APP_ID \
+  --client-secret $SP_PASSWORD \
```
Comment (jainmilind) [inline azure/onboarding/byok8s/manualsteps/README.md:51]: Declaration for `$SP_APP_ID` and `$SP_PASSWORD` is not added in doc above.

Comment (jainmilind) [reply azure/onboarding/byok8s/manualsteps/README.md:51]: Should we add definition for what all the variables mean here?

Comment (ritu-rubrik) [reply azure/onboarding/byok8s/manualsteps/README.md:51]: yeah missed this.. I will add that too to variables

File: azure/onboarding/byok8s/manualsteps/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+   Ensure this by executing the below command and selecting the right subscription.
+   ```
+   az login
+   ```
+3. Azure App Id and secret used to onboard exocompute feature in RSC via non-oauth flow.
+## Variables:
+1. RESOURCE_GROUP - This should be same as the resource group used to onboard exocompute as the permissions are taken at resource group level.
+2. VNET_NAME - The name of VNET where the AKS cluster is to be created. This VNET should meet all the network pre-requisites needed for exocompute clusters.
+3. AKS_SUBNET_NAME - Subnet to be used for the AKS cluster.
+4. VNET_RESOURCE_GROUP - resource group of the VNET where the cluster is created.
```
Comment (Sawarnik123) [inline azure/onboarding/byok8s/manualsteps/README.md:15]: `resource group of the VNET where the cluster is created.` -> `Resource group of the VNet.`

File: azure/onboarding/byok8s/manualsteps/README.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+5. SP_APP_ID - App Id used to onboard subscription for exocompute feature via non-Oauth flow in RSC.
+6. SP_PASSWORD - Client secret used while onboarding subscription for exocompute via non-Oauth flow in RSC.
+7. CLUSTER_NAME - Desired name of the BYOK cluster.
+8. LOCATION - Azure region where the cluster is to be created.
+   
+## Create Private AKS Cluster:
+Following are configuration we expect for the cluster.
+
+ - Starting node count with min-count of 1 and max-count of 64.
+ - Cluster auto scalar enabled.
```
Comment (Sawarnik123) [inline azure/onboarding/byok8s/manualsteps/README.md]: nit: `auto scalar` -> `autoscaler`

**Review by Sawarnik123 — APPROVED** [review]

**Review by jainmilind — APPROVED** [review]

**Review by arnavrupde1 — APPROVED** [review]

File: azure/onboarding/byok8s/manualsteps/README.md (hunk 4 above)
Comment (ritu-rubrik) [reply azure/onboarding/byok8s/manualsteps/README.md:15]: I think thsio gives more clarity that the cluster should be a part of same Vnet

**Review by RudraNirvan — APPROVED** [review]

**Pull request merged**